apps/buildog/src/app/organizations/[organizationId]/layout.tsx

```diff
@@ -12,7 +12,7 @@ export async function generateStaticParams() {
   }));
 }
 
-export default function Layout({
+export default async function Layout({
```

Comment:
Why this is async function?I believe async functions can usable with server components. Can you check please?

Also, we don't need async.